velox/functions/prestosql/Probability.h

```diff
@@ -15,6 +15,7 @@
  */
 #pragma once
 
+#include <boost/math/distributions/weibull.hpp>
```

Comment:
@Krishna-Prasad-P-V Do we need this import? Looks like all the computation is done with std.

Reply:
Thanks for reminding me. Removed the same.